src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -168,3 +168,17 @@ def _list_log_files_with_filter(  # pylint: disable=too-many-arguments
         files.append(f)
 
     return files
+
+
+#        Custom funcs for list servers        #
+def _server_list_custom_func(
```

Comment:
Thank @derekbekoe for your suggestion, but I saw the other functions all have the _ prefix, I am thinking keep the same naming rule is better.